dl/src/main/scala/com/intel/analytics/bigdl/optim/ValidationMethod.scala

```diff
@@ -124,10 +125,11 @@ class Top1Accuracy[T] extends ValidationMethod[T] {
 }
 
 class Top5Accuracy[T] extends ValidationMethod[T] {
-  override def apply(output: Activity, target: Activity, criterion: Criterion[T] = null):
+  override def apply(output: Activity, target: Activity):
   AccuracyResult = {
     val _output = output.asInstanceOf[Tensor[T]]
     val _target = target.asInstanceOf[Tensor[T]]
+    _target.squeeze()
```

Comment:
should not modify the original ground truth data. You may need to add a new squeeze() method which return a new tensor point to the same storage with a different view.